packages/dashboards/addon/templates/dashboards/dashboard/widgets/widget.hbs

```diff
@@ -100,10 +102,15 @@
           <CommonActions::Delete
             @model={{@model}}
             @deleteAction={{route-action "deleteWidget"}}
-            class="button is-text is-medium"
+            as |onDelete|
           >
-            <NaviIcon @icon="trash-o" class="navi-report-widget__icon" />
-            Delete
+            <DenaliButton
+              @style="text"
+              class="delete-button"
```

Comment:
```suggestion
              class="navi-report__delete-btn"
```

Reply:
wouldnt this be navi-report-widget__delete-btn?